profile/README.md

```diff
@@ -0,0 +1,9 @@
+Profiling
+---------
+###Prereqs
+
+`pip3 install snakeviz`
```

Comment:
Should add this as a dev dependency via `pipenv install --dev snakeviz` so it is persisted in the repo (and then add the changes to the pipfile / the lock). Then this README text could probably be added to the root readme.

Reply:
also to run it using pipenv..

`pipenv run snakeviz profile/<profilefilename>`

Reply:
done